### ISSUE TITLE
Version 0.8.25

### ISSUE THREAD

**Issue by beeb** [hellwolf/solc.nix#3]
Hello!

First of all thanks a lot for this overlay, I'm using it all the time and it's amazing!

Solidity 0.8.25 has been out for a couple of weeks and I was wondering if you were planning on updating this flake soon?

Cheers

**Comment by hellwolf** [comment]
I am very happy to hear that you have been benefiting from using it!

And it is done, logs:

```
$ ./utils/download.sh 0.8.25
$ ./utils/create-listing.sh 
$ git diff | more
diff --git a/solc-listing.nix b/solc-listing.nix
index c99f5d2..2caa34e 100644
--- a/solc-listing.nix
+++ b/solc-listing.nix
@@ -1,4 +1,12 @@
 [ # DO NOT MODIFY! AUTO GENERATED BY ./utils/create-listing.sh
+{
+  version = "0.8.25";
+  sha256 = {
+    solc-static-linux = "sha256-xCqtp6UgV92+2T7AESNeJWxWTEQLaNuqxa5IK6u7PW0=";
+    solc-macos = "sha256-zD+UpwrGgbAwQISswZgKq+KhuzJA1Eznao3w4ed6IRA=";
+    # solc-macos-aarch not available for this version
+  };
+}
 {
   version = "0.8.24";
   sha256 = {
@@ -572,7 +580,7 @@
   sha256 = {
     solc-static-linux = "sha256-iYpeBdOsCNcmsBnSQVwcCHqa6HhmzfpVHAiza2tFMh0=";
     # solc-macos not available for this version
-    # solc-macos-aarch not available for this version
+    solc-macos-aarch = "sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=";
   };
 }
 {
diff --git a/utils/download.sh b/utils/download.sh
index 6d1bd7d..be06eee 100755
--- a/utils/download.sh
+++ b/utils/download.sh
@@ -8,7 +8,7 @@ list_all_versions() {
     for i in $(seq  0 17); do echo 0.5."$i"; done
     for i in $(seq  0  9); do echo 0.6."$i"; done
     for i in $(seq  0  6); do echo 0.7."$i"; done
-    for i in $(seq  0 24); do echo 0.8."$i"; done
+    for i in $(seq  0 25); do echo 0.8."$i"; done
 }
 
 download_one_version() {
$ git commit -m 'included solc 0.8.25' .
```

**Comment by beeb** [comment]
Thank you!
